.github/workflows/e2e.yml

```diff
@@ -28,30 +28,53 @@ jobs:
         with:
           submodules: true
           path: vip-go-mu-plugins
+
       - name: Setup Node
         uses: actions/setup-node@v2
         with:
           node-version: '16'
+          cache: npm
+          cache-dependency-path: vip-go-mu-plugins/package-lock.json
+
       - name: Install dependencies
-        run: cd ./vip-go-mu-plugins && npm install
-      - name: Install VIP Cli
-        run: npm install --prefix=$HOME/.local -g @automattic/vip
+        run: npm ci
```

Comment:
Use `npm ci` instead of `npm install` to ensure that we won't install any new package versions; only what is set in `package-lock.json` will be installed.